src/preferences/configobject.cpp

```diff
@@ -220,7 +220,7 @@ template<class ValueType>
 QSet<QString> ConfigObject<ValueType>::getGroups() {
     QWriteLocker lock(&m_valuesLock);
     QSet<QString> groups;
-    for (const ConfigKey& key : m_values.uniqueKeys()) {
+    for (const ConfigKey& key : m_values.keys()) {
```

Comment:
Fun fact: this was my original attempt. https://github.com/mixxxdj/mixxx/pull/2822#discussion_r431446829
Thanks for fixing it.